android/src/main/java/com/tailscale/ipn/ui/viewModel/MainViewModel.kt

```diff
@@ -53,6 +55,7 @@ class MainViewModel : IpnViewModel() {
         it?.let { netmap ->
           peerCategorizer.regenerateGroupedPeers(netmap)
           peers.set(peerCategorizer.groupedAndFilteredPeers(searchTerm.value))
+          isAdmin.set(netmap.SelfNode.isAdmin)
```

Comment:
This doesn't quite work. For example, on tailscale.com, I don't have `https://tailscale.com/cap/is-admin`, but I am able to approve the device because I'm an [IT Admin](https://tailscale.com/kb/1138/user-roles#it-admin). I should see the link, but I don't.

Reply:
AFAICT, we don't have a way of knowing whether or not someone is an IT Admin. Given that, I'd probably err on the side of showing the link.

Reply:
I guess we never considered that. The cap/is-admin came first and then we added new admin types later:

```
commit 6a87f4dd42bdd547ef6363ac4beb07920adf55e7
Author: Brad Fitzpatrick <...>
Date:   Fri Apr 9 08:25:24 2021 -0700

    control: tell nodes in MapResponse whether they're an admin


commit 15bb5e2adaf7e09af2e838f3fb5e484cb759261c
Author: Sonia Appasamy <...>
Date:   Fri Nov 5 17:16:52 2021 -0400

    control/cfgdb,adminhttp: add new admin console roles
    
    This commit adds three new admin console roles to domains: Network
    Admin, IT Admin, and Auditor. This change also uses these new roles
    to permission what different users can access from the admin api.
    
    Change-Id: Ib4143842699c2d9e74b7f219b8927a8ef30dc6a4
```

BTW, what we'd want to advertise to nodes is "can you authorized a device" and not "are you an IT admin".

The whole "IT Admin" vs "Network Admin" roles I kinda view as a mistake. They conflate titles and capabilities. Maybe they should've been templates people could then rename & tweak permissions on. (Maybe we'll do that later). Related: https://tailscale.com/blog/rbac-like-it-was-meant-to-be